yt_dlp/extractor/newgrounds.py

```diff
@@ -88,10 +92,10 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, media_id)
 
         title = self._html_search_regex(
-            r'<title>([^>]+)</title>', webpage, 'title')
+            r'<title>(.+?)</title>', webpage, 'title')
```

Comment:
why?

Reply:
Title could contain `>` character and not be detected by yt-dlp.

Reply:
That's not valid HTML. It would be escaped as `&gt;`.

Reply:
I didn't know that, still gonna leave it doesn't break anything.

Reply:
if the webpage contains multiple title tags, it could